pkg/model/cluster.go

```diff
@@ -130,7 +135,7 @@ func NewCluster(env simulator.Environment, config ClusterConfig, replicasConfig
 	cm := &clusterModel{
 		env:                 env,
 		config:              config,
-		replicasConfig:		 replicasConfig,
```

Comment:
Nit: replace tabs with spaces.  Tip: setup your IDE to run `gofmt` on every save you never have to think about formatting.   :)

Reply:
Set up my IDE.